README.md

```diff
@@ -1,26 +1,24 @@
 # XLT
 XLT is an extensive load and performance test tool developed and maintained by Xceptance. If you need more information, here is the current website https://www.xceptance.com/xlt/ and the current documentation https://xltdoc.xceptance.com/. There is also a forum available to discuss load testing and test automation with XLT: https://ask.xceptance.de/
 
-The documentation was fully rewritten in 2020 and it is continuously updated. We appreciate your feedback. You can also directly contribute on GitHub https://github.com/Xceptance/xlt-documentation. If you need any information from the legacy documentation, you can still find it at https://lab.xceptance.de/releases/xlt/latest/index.html
+The documentation was fully rewritten in 2020 and it is continuously updated. We appreciate your feedback. You can also directly contribute on GitHub at https://github.com/Xceptance/xlt-documentation. If you need any information from the legacy documentation, you can still find it at https://lab.xceptance.de/releases/xlt/5.7.1/index.html
 
 # Open Source
-XLT has been open sourced on 30 January 2020 under the Apache License 2.0. Stay tuned for more!
+XLT has been open sourced on 30 January 2020 under the Apache License 2.0.
 
 # How to Build XLT
 
 The XLT build process is based on the **Apache Ant** build tool. The build works best when being run on a Linux machine.
 
 ## Required Tools
 
-* Latest JDK 8 (or 11)
```

Comment:
Was support for building on JDK 11 removed?

Reply:
No, but we do not test that regularly since we build with Java 8 only. Proper Java 11 support comes (back) with the other merge request.